src/tbb/arena.h

```diff
@@ -397,158 +372,58 @@ class arena: public padded<arena_base>
 
     std::uintptr_t calculate_stealing_threshold();
 
+    unsigned priority_level() { return my_priority_level; }
+
+    int has_request() { return my_total_num_workers_requested; }
+
+    unsigned references() { return my_references.load(std::memory_order_acquire); }
+
+    bool is_arena_workerless() { return my_max_num_workers == 0; }
+
+    bool is_top_priority();
```

Comment:
Consider making these methods (and similar ones) to possess constness:
```suggestion
    unsigned references() const { return my_references.load(std::memory_order_acquire); }

    bool is_arena_workerless() const { return my_max_num_workers == 0; }

    bool is_top_priority() const;
```